modules/sdk-coin-sol/src/lib/utils.ts

```diff
@@ -330,6 +333,27 @@ export function validateIntructionTypes(instructions: TransactionInstruction[]):
   }
 }
 
+/**
+ * Validate solana instructions match raw msg authorize transaction
+ *
+ * @param {TransactionInstruction} instructions - a solana instruction
+ * @returns {boolean} true if the instructions match the raw msg authorize transaction
+ */
+export function validateRawMsgInstruction(instructions: TransactionInstruction[]): boolean {
+  // as web3.js cannot decode authorize instruction from CLI, we need to check it manually first
+  if (instructions.length === 2) {
+    const programId1 = instructions[0].programId.toString();
+    const programId2 = instructions[1].programId.toString();
+    if (programId1 === SystemProgram.programId.toString() && programId2 === StakeProgram.programId.toString()) {
+      const instructionName1 = SystemInstruction.decodeInstructionType(instructions[0]);
+      const data = instructions[1].data.toString('hex');
+      if (instructionName1 === 'AdvanceNonceAccount' && data === '0a00000001000000') {
```

Comment:
Please define these strings somewhere. (or use a value exposed by the library)

Reply:
I see that the constants were defined, but still not used here